homework3/semi-uber/Server/api/handlers/trucksHandlers/trucksDeleteHandler.js

```diff
@@ -5,15 +5,23 @@ const errorHandler = require("../error.handler");
 
 const trucksDeleteHandler = (req, res) => {
   const truckID = req.headers["truck"];
-  Truck.findOneAndRemove({ _id: truckID, assigned_to: null })
-    .then(() => logger.info("Truck was successfully deleted"))
-    .then(() =>
-      res.json({
-        success: true,
-        data: null,
-        error: null
-      })
-    )
+
+  Truck.findOneAndRemove({ _id: truckID, assigned_to: null, edit: true })
+    .then(result => {
+      if (result === null) {
+        errorHandler(
+          "Profile info can not be edited, when some of trucks is status 'OL'",
+          res
+        );
+      } else {
```

Comment:
i think, i don't need else  here, if i gonna use 'return'  at string 12

Reply:
As for me it is better lo leave it as it is. Because in fact, `errorHandler` function is not intended to return something